### PR TITLE
fix: Unable to find JSON Schema ID: default

### PR DESCRIPTION
**Pull request by splaunov**
When creating identity with admin api the following error is generated:

```
Unable to find JSON Schema ID: default

[github.com/ory/kratos/schema.Schemas.GetByID](http://github.com/ory/kratos/schema.Schemas.GetByID)
/go/src/github.com/ory/kratos/schema/schema.go:37 
[github.com/ory/kratos/identity.(*Validator](http://github.com/ory/kratos/identity.(*Validator)).ValidateWithRunner
/go/src/github.com/ory/kratos/identity/validator.go:40 
[github.com/ory/kratos/identity.(*Validator](http://github.com/ory/kratos/identity.(*Validator)).Validate 
```

This happens if there is no schema id `default` in the config.

## Checklist

- [ x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ x] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ x] I have read the [security policy](../security/policy).
- [ x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: identity/manager.go [hunk 1]
```diff
@@ -62,6 +64,10 @@ func newManagerOptions(opts []ManagerOption) *managerOptions {
 }
 
 func (m *Manager) Create(ctx context.Context, i *Identity, opts ...ManagerOption) error {
+	if i.SchemaID == "" {
```
Comment (aeneasr) [inline identity/manager.go:67]: Could you please provide a failing test case for the issue, which is then resolved by this change? That will help with any future regressions :)

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: identity/manager.go (hunk 1 above)
Comment (aeneasr) [reply identity/manager.go:67]: ping @splaunov

Comment (splaunov) [reply identity/manager.go:67]: Have added the test

**Review by aeneasr — APPROVED** [review]
Thank you so much for following up :))

**Pull request merged**